packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -37,7 +37,7 @@
     "@babel/plugin-transform-shorthand-properties": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-beta.4200.gf3d0a27",
-    "@okta/courage": "4.5.0-7417-gee15656",
+    "@okta/courage": "4.5.0-7446-g3dbc3d5",
```

Comment:
- [ ] version to be updated after merge https://github.com/okta/okta-ui/pull/5867